drivers/include/pcf857x.h

```diff
@@ -0,0 +1,542 @@
+ *   pulled up to HIGH high by a weak 100 μA pull-up, which corresponds to the
+ *   behavior of the #GPIO_IN_PU and #GPIO_OD_PU modes.
+ * - OUTPUT:
+ *   Writing a 0 to an expander pin configures the pin as an OUTPUT and
+ *   actively drives the pin to LOW, which corresponds to the behavior of
+ *   the #GPIO_OUT and #GPIO_OD_PU modes.
+ *
+ * @note The driver therefore supports the #GPIO_IN_PU and #GPIO_OD modes as
+ * well as the physically identical modes #GPIO_IN, #GPIO_OUT and #GPIO_OD_PU.
+ * Please keep this in mind when connecting these pins to other open-drain
+ * or output pins that do not generate active signals.
+ *
+ * After the initialization with the #pcf857x_init function, all
+ * expander I/O pins are in input mode and pulled-up to HIGH.
+ *
+ * The expander I/O pins can be adressed as GPIO pins using the following
+ * scheme:
+ *
+ * PCF857X pin label | Port | Pin | RIOT symbol               | Remark
+ * ----------------- |:----:|:---:|:--------------------------|:-----------------
+ * P00               | 0    | 0   | `PCF857X_GPIO_PIN(0, 0)`  | PCF8574, PCF8574A and PCF8575
+ * P01               | 0    | 1   | `PCF857X_GPIO_PIN(0, 1)`  | PCF8574, PCF8574A and PCF8575
+ * ...               | ...  | ... | ...                       | ...
+ * P07               | 0    | 7   | `PCF857X_GPIO_PIN(0, 7)`  | PCF8574, PCF8574A and PCF8575
+ * P10               | 0    | 8   | `PCF857X_GPIO_PIN(0, 8)`  | PCF8575 only
+ * P11               | 0    | 9   | `PCF857X_GPIO_PIN(0, 9)`  | PCF8575 only
+ * ...               | ...  | ... | ...                       | ...
+ * P17               | 0    | 15  | `PCF857X_GPIO_PIN(0, 15)` | PCF8575 only
+ *
+ * ## Interrupts
+ *
+ * PCF857X expander has an open-drain, low-active interrupt (INT) signal, which
+ * generates an interrupt by any rising or falling edge of the expander pins
+ * in the input mode. Using this expander interrupt signal, the following
+ * features become available:
+ *
+ * - An interrupt service function can be attached to an expander input pin with
+ *   the #pcf857x_gpio_init_int function. This interrupt service function is
+ *   then called on any rising and/or falling edge of the expander input pin.
+ *
+ * - The driver can keep an internal up-to-date state of all expander input
+ *   pins. Using this internal up-to-date state of all expander input pins
+ *   avoids to read all expander input pins via I2C each time a single input
+ *   value of an expander GPIO pin is read with #pcf857x_gpio_read.
+ *
+ * Since interrupts are handled in the context of a separate event thread,
+ * enabling interrupts requires more RAM. Therefore, interrupts have to be
+ * activated explicitly with the module `pcf857x_irq`. Furthermore, the MCU
+ * interrupt pin to which the PCF857X INT signal is connected has to be
+ * defined by the parameter pcf857x_params_t::int_pin either in the parameter
+ * file or at the command line, e.g.
+ *
+ *      CFLAGS="-DPCF857X_PARAM_INT_PIN=\(GPIO\(0,6\)\)" \
```

Comment:
Is it possible you meant GPIO**_PIN**(0,6) here?

Reply:
Indeed, thanks for pointing that. I will fix it, when I'm back from vacation.